.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
Do we want to put the dist here? I guess that means we need to remember updating it all the time.